File: test/fixtures/dart/get_follow_redirect.dart
import 'package:http/http.dart' as http;

void main() async {
  var url = Uri.parse('http://localhost:28139');
  var res = await http.get(url);
  if (res.statusCode != 200) throw Exception('http.get error: statusCode= ${res.statusCode}');
  print(res.body);
}

File: test/fixtures/rust/get_follow_redirect.rs
extern crate reqwest;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let client = reqwest::blocking::Client::new();
    let res = client.get("http://localhost:28139")
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}
